utils/rollup-version-revision.mjs

```diff
@@ -5,9 +5,15 @@ import { execSync } from 'node:child_process';
  * @returns {string} Version string like `1.58.0-dev`
  */
 function getVersion() {
-    const text = readFileSync('./package.json', 'utf8');
-    const json = JSON.parse(text);
-    return json.version;
+    let version;
+    try {
+        const root = execSync('git rev-parse --show-toplevel').toString().trim();
+        const pkg = JSON.parse(readFileSync(`${root}/package.json`, 'utf8'));
+        version = pkg.version;
+    } catch (e) {
+        version = '0.0.0';
```

Comment:
Wouldn't it be more accurate to say "Unknown Version" or just don't show it at all?